modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[id='op-using-custom-scc-to-improve-prod-security_{context}']
+= Using a custom SCC to improve pod security
+
+Improve the security of your builds and runs by using custom security context constraints (SCCs).
+
+. Procedure
+
+* Edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext`.
```

Comment:
```suggestion
* In the `pipelines-scc` YAML file, modify the `runAsUser` and `seLinuxContext` elements as shown in the following example.
```
Use labels like `file` or `elements` to identify the type of object you're referencing.